### PR TITLE
Minimal

### PR DESCRIPTION
**Pull request by guillaumemoby**
Before sending your pull-request 💌 make sure :

- [x] The tests are up-to-date and your code is tested

Fixes https://github.com/disc-in/UltimateApp/issues/238

### CODE REVIEW

File: src/Components/DrillPageMinimal.js [hunk 1]
```diff
@@ -42,27 +43,35 @@ export const DrillPageMinimal = (props) => {
...
+          {I18n.t('drillPageMinimal.headerTitle', { trainingTitle: drill.title })}
         </Text>
-        <ProgressBar total={training.drills.length} current={currentDrillIndex + 1} onDotPress={onProgressDotPress} />
+        <View >
+          <ProgressBar total={training.drills.length} current={currentDrillIndex + 1} onDotPress={onProgressDotPress} />
+        </View>
       </View>
     );
+
+    const headerRight = () => {
```
Comment (sallesma) [inline src/Components/DrillPageMinimal.js:54]: I think we decided (you wrote this in the issue #238 ) to remove this button and put the description at the bottom of the screen ?

File: src/Components/DrillPageMinimal.js [hunk 2]
```diff
@@ -112,7 +121,7 @@ const styles = StyleSheet.create({
     marginLeft: '10%',
     marginTop: 15,
     marginBottom: 5,
-    width: '80%',
```
Comment (sallesma) [inline src/Components/DrillPageMinimal.js:115]: I think the 80% was here to handle long titles

For example:

Capture du 2020-11-12 18-05-22


Some people have smaller phones than we do, and they might have the problem even if we don't have it

File: src/Components/DrillPageMinimal.js (hunk 1 above)
Comment (guillaumemoby) [reply src/Components/DrillPageMinimal.js:54]: I havn't succeed to to it. The other problem is the scrolldown on ios doesn't work well . We should change it

**Comment by guillaumemoby** [comment]
It's working on Ios

**Review by sallesma — APPROVED** [review]

**Pull request merged**